qiita_pet/static/js/sampleTemplateVue.js

```diff
@@ -60,6 +60,7 @@ Vue.component('sample-template-page', {
               '</div>' +
             '</div>',
   props: ['portal', 'study-id'],
+  maxDirectUploadSize: 2097152,
```

Comment:
2MB? Any reason to keep it so limited?

Reply:
Comes from original issue discussion on the file sizes: https://github.com/biocore/qiita/issues/2433; I can raise if you think is fine/safe; do you have a # suggestion?

Reply:
At least 10MB given EBI's limit?